other/restrict-clusteradmin-rolebindings-gcp-2024-003/restrict-clusteradmin-rolebindings-gcp-2024-003.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: restrict-clusteradmin-rolebindings-gcp-2024-003
+  annotations:
+    policies.kyverno.io/title: Restrict ClusterAdmin RoleBindings (GCP-2024-003)
+    policies.kyverno.io/category: Security
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: RoleBinding, ClusterRoleBinding
+    kyverno.io/kubernetes-version: 1.27
+    kyverno.io/kyverno-version: 1.10
+    policies.kyverno.io/description: >-
+      The cluster-admin ClusterRole allows any action to be performed on any resource in the cluster and its granting should be heavily restricted. This policy prevents binding to the cluster-admin ClusterRole in RoleBinding or ClusterRoleBinding resources to the system:authenticated, system:unauthenticated, and system:anonymous subjects.
+
+      Refer to the documentation for more details on Kyverno annotations: https://artifacthub.io/docs/topics/annotations/kyverno/
+spec:
+  validationFailureAction: Audit
+  rules:
+  - name: validate-restricted-cluster-admin-bindings
+    match:
+      resources:
+        kinds:
```

Comment:
Match and exclude blocks need to use `any` or `all` to be compatible with the current schema.